crates/biome_js_analyze/src/lint/nursery/use_consistent_member_accessibility.rs

```diff
@@ -0,0 +1,558 @@
+    /// ```jsx,ignore
+    /// class Animal {
+    ///   constructor(
+    ///     public breed,
+    ///     name,
+    ///   ) {
+    ///     // Parameter property and constructor
+    ///     this.animalName = name;
+    ///   }
+    ///   private animalName: string; // Property
+    ///   get name(): string {
+    ///     // get accessor
+    ///     return this.animalName;
+    ///   }
+    ///   set name(value: string) {
+    ///     // set accessor
+    ///     this.animalName = value;
+    ///   }
+    ///   protected walk() {
+    ///     // method
+    ///   }
+    /// }
+    /// ```
+    ///
+    /// The following patterns are considered correct code with the accessibility set to `explicit`:
+    /// ```jsx,ignore
+    /// class Animal {
+    ///   public constructor(
+    ///     public breed,
+    ///     name,
+    ///   ) {
+    ///     // Parameter property and constructor
+    ///     this.animalName = name;
+    ///   }
+    ///   private animalName: string; // Property
+    ///   public get name(): string {
+    ///     // get accessor
+    ///     return this.animalName;
+    ///   }
+    ///   public set name(value: string) {
+    ///     // set accessor
+    ///     this.animalName = value;
+    ///   }
+    ///   protected walk() {
+    ///     // method
+    ///   }
+    /// }
+    /// ```
+    ///
+    /// The following patterns are considered correct code with the accessibility set to `none`:
+    /// ```jsx,ignore
+    /// class Animal {
+    ///   constructor(
+    ///     breed,
+    ///     name,
+    ///   ) {
+    ///     // Parameter property and constructor
+    ///     this.name = name;
+    ///   }
+    ///   animalName: string; // Property
+    ///   get name(): string {
+    ///     // get accessor
+    ///     return this.animalName;
+    ///   }
+    ///   set name(value: string) {
+    ///     // set accessor
+    ///     this.animalName = value;
+    ///   }
+    ///   walk() {
+    ///     // method
+    ///   }
+    /// }
+    /// ```
+    ///
+    /// ## Options
+    ///
+    /// The rule supports the following options:
+    ///
+    /// ```json
+    /// {
+    ///     "//": "...",
+    ///     "options": {
+    ///         "accessibility": "explicit"
+    ///     }
+    /// }
+    /// ```
+    ///
+    /// ### `accessibility`
+    /// This option determines the required accessibility modifiers on class properties and methods.
+    /// It can be set to one of the following values:
+    /// - `noPublic` - forbid the use of public (a safe fix will remove it).
+    /// - `explicit` - requires an accessibility modifier for every member that allows that (a safe fix will add public).
+    /// - `none` - forbid all accessibility modifiers (public, protected, private).
+    ///
+    /// Default: `noPublic`.
+    ///
+    pub UseConsistentMemberAccessibility {
+        version: "1.8.0",
+        name: "useConsistentMemberAccessibility",
+        language: "js",
```

Comment:
This should be `ts` right?
```suggestion
        language: "ts",
```